src/Svelte/doc/index.rst

```diff
@@ -41,10 +41,10 @@ Next, install a package to help Svelte:
 
 .. code-block:: terminal
 
-    $ npm install svelte svelte-loader --save-dev
+    $ npm install svelte-loader --save-dev
 
     # or use yarn
-    $ yarn add svelte svelte-loader --dev
+    $ yarn add svelte-loader --dev
```

Comment:
`svelte` itself is no longer needed in the user's project?

Reply:
It is automatically added when you install `symfony/ux-svelte`, I think I removed it to be consistent with the other docs